env.md

```diff
@@ -0,0 +1,27 @@
+# Environment variables
```

Comment:
Split environment variables and Slack OAuth configuration into separate files. This is the first of those.